store/tikv/client.go

```diff
@@ -564,24 +564,26 @@ func sendBatchRequest(
 		err:      nil,
 	}
 	ctx1, cancel := context.WithTimeout(ctx, timeout)
```

Comment:
We should keep defer, the new go version call allocate the defer OP on stack. See https://go-review.googlesource.com/c/go/+/171758/